zeeguu/api/endpoints/translation.py

```diff
@@ -89,6 +91,7 @@ def get_one_translation(from_lang_code, to_lang_code):
             to_lang_code,
             context,
             article_id,
+            learning_cycle=LearningCycle.RECEPTIVE if _merle_exercises(flask.g.user) else LearningCycle.NOT_SET,
```

Comment:
The problem here is: what is the meaning of this column? Learning cycle? Is it the current learning cycle? If it has not been scheduled yet, then I think it's unfair to say that it's in either of the cycles. 

Thus, what I would propose is that here, you initialize *all* of the Bookmarks with `NOT_SET` and then only the first time a word is seen in an exercise, this column gets to be set. Do you see what I mean?

Reply:
Alternatively, let's leave it like this for now. Then when @tfnribeiro merges his own changes that pertain the exercises, he makes this change himself. He already has written code for detecting that moment when a bookmark is entering the "studying pipeline".